### DISCUSSION TITLE
Turning individual operations on/off

### DISCUSSION THREAD

**Discussion by mkiang** [redivis/meta#180] [Feature requests]
It would be super useful (for debugging) if we could turn on/off individual operations. Such as below. Right now to debug a `resources exceeded` error, I need to copy the whole transform and delete individual operations but it would be nice to be able to toggle them on/off instead. 

<img width="772" alt="Screen Shot 2021-01-04 at 3 15 56 PM" src="https://user-images.githubusercontent.com/3435045/103589210-c8c43300-4e9f-11eb-98fc-c6b4696c284d.png">

**Comment by sfmcintyre** [comment]
thanks for the note, Matt, toggling individual blocks seems very useful. we're working on a number of transform updates in this vein as we speak, including more modular, copy/paste-able transform operations, and we'll definitely consider toggling & hiding in this new block design. i imagine an improved variable statistics interface (with more robust statistics, perhaps customizable) would be helpful for this debugging use case - this is also on the docket, though a little further out

**Comment by sfmcintyre** [comment]
Implemented in our [latest release](https://redivis.com/updates#2023-06-22)
